CONTRIBUTING.md

```diff
@@ -0,0 +1,267 @@
+<!---
+Copyright 2020 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# How to contribute to huggingface_hub, the GitHub repository?
+
+Everyone is welcome to contribute, and we value everybody's contribution. Code
+is thus not the only way to help the community. Answering questions, helping
+others, reaching out and improving the documentations are immensely valuable to
+the community.
+
+It also helps us if you spread the word: reference the library from blog posts
+on the awesome projects it made possible, shout out on Twitter every time it has
+helped you, or simply star the repo to say "thank you".
+
+Whichever way you choose to contribute, please be mindful to respect our
+[code of conduct](https://github.com/huggingface/huggingface_hub/blob/master/CODE_OF_CONDUCT.md).
+
+## You can contribute in so many ways!
+
+The repository is split into different parts, where we welcome contributions.
+
+### The client library, `huggingface_hub`
+
+This repository hosts the client library `huggingface_hub`, which is a frontend to the Hugging Face Hub.
+this part lives in `src/huggingface_hub` and `tests`.
+
+There are many ways you can contribute to this client library:
+* Fixing outstanding issues with the existing code;
+* Contributing to the examples or to the documentation;
+* Submitting issues related to bugs or desired new features.
+
+When opening a PR on this part of the repository, please add as reviewer:
+- @LysandreJik
+- @osanseviero
```

Comment:
What do we think of automatically adding them as a reviewer, by listing you both in a CODEOWNERS file? 

We can then also even do the specific owners of parts of the library, so if it was touched automatically myself or Steven are put on as reviewers. No need for them to have to remember to do this.

Reply:
Addressed with: https://github.com/huggingface/huggingface_hub/pull/723

@LysandreJik I think this part can be completely taken out now with this as well, since it's automatically done.